videos/nappn_2021/walkthrough.Rmd

```diff
@@ -0,0 +1,340 @@
+    - Download with R using code below
+
+```{r, eval=FALSE}
+local_path <- "dryad_data.zip"
+download.file(url = "https://datadryad.org/stash/downloads/file_stream/624637", destfile = local_path)
+unzip(local_path)
+```
+
+## Data overview
+
+The purpose of this walkthrough is to provide an introduction the TERRA REF public domain high throughput phenomics datasets (LeBauer et al 2020), and to give learners a chance to access and explore a representative cross section of these data. 
+
+In addition to this workshop, we have online tutorials for using these data at [https://terraref.github.io/tutorials/](https://terraref.github.io/tutorials/). There are also materials from previous walkthrough workshops at [https://github.com/terraref/tutorials/tree/master/videos](https://github.com/terraref/tutorials/tree/master/videos). We are striving to make these data as accessible and usable as possible. 
+
+TERRA REF is a project funded by the Advanced Research Projects Agency for Energy (ARPA-E). The objective is to advance the science and technology of high throughput field phenotyping, which is the automated measurement of plants. TERRA REF has deployed an exceptionally large robot capable of collecting data from a range of sensors at 1 mm spatial resolution every other day. You can learn more about the project from the website, terraref.org. The suite of sensors include visible light cameras, hyperspectral cameras, a laser 3d scanner, and environmental data. In addition there are weather stations and high resolution genome sequences for hundreds of varieties of Sorghum being monitored. 
+
+### Experimental design
+
+(Show picture of site) Field site is in Arizona. Field is passed over by large robot called a gantry. Lot of equipment in hanging box to collect many types of data. 
+
+Gantry goes systematically over entire field once a day. Some sensors take data every day, like 9,000 images from RGB camera. Others more intermittently, like hyperspectral images, because of data space limits. 
+
+Sensors include: 
+
+* camera that takes pairs of red-green-blue images (**Stereo RGB**)
+* thermal infrared images (**FLIR**)
+* images at a bunch of wavelengths to get hyperspectral data (**VNIR/SWIR**)
+* laser that collects points on plant surfaces to create 3D image (**3D Laser**)
+* measures plant fluorescence (**PS II Fluor**)
+* handful of others, including environmental data such as temperature and light
+
+Collecting data since 2015, and are up to 8 seasons worth in that time. Originally for sorghum, but now open to other crop species and organizations that want to use system.
+
+### Traitvis webapp
+
+The easiest way to start to visualize and understand the TERRA REF data is through our webapp. Go to [https://traitvis.workbench.terraref.org/](https://traitvis.workbench.terraref.org/), which takes a minute to load. Displays plots of various data across collection time.
```

Comment:
```suggestion
The easiest way to start to visualize and understand the TERRA REF data is through the Traitvis shiny app website. Go to [terraref.org/traitvis/](https://terraref.org/traitvis/); this takes a minute to load because there is so much data, and _may freeze if everyone does it at once_! It displays time plots of trait data as well as maps and thumbnails of field scans.
```

Reply:
I think whether or not we encourage attendees to actually go to the traitvis webapp themselves depends on how many people there are? If there are a lot of people, we should recommend they just watch and go to the site some time later.